src/test/java/com/wikia/webdriver/testcases/desktop/spamwikireview/ViewSpamWikisTests.java

```diff
@@ -0,0 +1,77 @@
+package com.wikia.webdriver.testcases.desktop.spamwikireview;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.communities.desktop.pages.SpamWikiReviewPage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+@Test(groups = "SpamWikiReview")
+public class ViewSpamWikisTests extends NewTestTemplate {
+
+  @Test
+  @Execute(asUser = User.STAFF)
+  public void openSpamWikiReviewTest() {
+    SpamWikiReviewPage spamWikiReviewPage = new SpamWikiReviewPage();
+    spamWikiReviewPage.open();
+
+    //TODO: assert here that it was opened
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)